massa-models/src/config/constants.rs

```diff
@@ -62,7 +62,7 @@ lazy_static::lazy_static! {
     /// node version
     pub static ref VERSION: Version = {
         if cfg!(feature = "sandbox") {
-            "SAND.0.0"
+            "SAND.0.20"
```

Comment:
```suggestion
            "SAND.20.0"
```

It would be better to put the corresponding testnet version and update the compatibility check since this is going to be merge into the release branch

Reply:
Done